libs/ngx-dsfr/header/src/lib/header.component.scss

```diff
@@ -5,6 +5,84 @@
 
 // Actual styling
 .fr-header {
+  &.mini-mobile-header {
+    .fr-header__brand {
+      flex-wrap: nowrap;
+
+      .fr-header__brand-top {
```

Comment:
The `width: auto;` for `.fr-header__brand-top` that I gave you in my comment is actually necessary, otherwise the app name will not be close to the Marianne logo